### DISCUSSION TITLE
How to run burgr033's autotiling script when GlazeWM is started?

### DISCUSSION THREAD

**Discussion by mizzunet** [glzr-io/glazewm#853] [General]
This one https://github.com/burgr033/GlazeWM-autotiling-python/tree/main.
I tried this,
```
general:
  startup_commands: [ 'python3 C:\Users\Missu\.glaze-wm\main.py' ]
```

That did not start the script. But if I run the script manually, it works.

Thanks

**Comment by oligocene** [comment]
I had a similar (maybe?) problem. Ended up installing Python to a non-standard location (to avoid path/spaces problems), then starting the script via a batch file.

Batch file:
`START /MIN C:\Python3\python.exe C:\Scripts\autotiling.py`

In config:

`startup_commands: [ 'C:\Scripts\autotiling.cmd' ]`

Works for me.

**Reply by mizzunet** [reply]
I'm afraid, this doesn't work for me. Also, running this cmd script manually runs the python in the foreground rather than backgroud
